src/funcs.rs

```diff
@@ -191,9 +191,10 @@ pub fn random_footer<'a>(
 ) -> Cow<'a, str> {
     match rand::thread_rng().gen_range(0..4) {
```

Comment:
This should be `0..=4` or `0..5` otherwise the last branch is unreachable.

Reply:
Oh yea my bad I will add that. Thank you!